cssom-1/Overview.bs

```diff
@@ -71,6 +71,9 @@ urlPrefix: https://html.spec.whatwg.org/multipage/
     urlPrefix: semantics.html
         type: dfn
             text: a style sheet that is blocking scripts
```

Comment:
This is still used by the `Link` section, so we can probably remove it once we address https://github.com/w3c/csswg-drafts/pull/8759#issuecomment-1523008142.